ipatests/test_integration/test_commands.py

```diff
@@ -1235,6 +1235,21 @@ def test_cb(stdout):
         cmd = ["journalctl", "-u", "sshd", f"--since={since}"]
         tasks.run_repeatedly(self.master, command=cmd, test=test_cb)
 
+    def test_user_del_preserve(self):
+        """
+        Test if preserving a user account produces correct output message.
+
+        https://pagure.io/freeipa/issue/9187
+        """
+        user = "testuser"
```

Comment:
before adding or deleting users, you have to `kinit` an administration user. In this case using `tasks.kinit_admin(self.master)`